planetarium/NineChronicles
# Fix bug in reconnecting rpc connection.

### Description

1. Fix exceptions/bugs during retrying rpc connection.

### How to test

1. Launch game via remote headless mode.
2. Restart currently connected node.
3. Check if connection is correctly handling. (timeout, reconnection, etc...)

### Related Links
https://app.asana.com/0/958521740385861/1201683861701348

### Required Reviewers
@planetarium/9c-dev
